common_patches/i2c0.yaml

```diff
@@ -0,0 +1,28 @@
+COMD%s:
+    _delete: ["COMMAND"]
+    _add:
+      OPCODE:
+        description: "Opcode part of command %s. 0: RSTART, 1: WRITE, 2: READ, 3: STOP, 4: END."
+        bitOffset: 11
+        bitWidth: 3
+        access: read-write
+      ACK_VALUE:
+        description: "Acknowledge value for command %s."
```

Comment:
`svd2rust` doesn't know how to take "%s" for field description from register array index.
Should it? If yes, how it should look like?

Reply:
It doesn't have to process it in any way, this is done in different arrays within our crate just to somehow reflect in the description that we are working with an array (even though it's not necessary).

You have also included `%s` in the description section multiple times, e.g. [here](https://github.com/esp-rs/esp-pacs/blob/main/common_patches/ledc_int.yaml#L5-L42). Don't see any problem here, sorry.